helpers/l10n.ts

```diff
@@ -17,49 +17,27 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import { toNotSoISOString } from './dates';
-import { getJSON } from './request';
-import { get, save } from './storage';
+import { getL10nLocale, getL10nMessages } from './init';
```

Comment:
Maybe it is just me but `L10n` brings noise to the variables and functions names. WDYT ?

Reply:
True, I'll drop the L10n part!